detection-rules/attachment_office_external_template.yml

```
@@ -0,0 +1,29 @@
+name: "Attachment: Office document loading external template"
+description: |
+  Recursively scans archives and Office documents to detect external templates, this technique is used in vulnerabilities such as CVE-2022-30190 (Follina)
+type: "rule"
+severity: "medium"
+source: |
+  type.inbound
+  and any(attachments,
+          (
+            // office files
+            .file_extension in~ $file_extensions_macros
+            or .file_extension in~ $file_extensions_common_archives
+          )
+          and file.oletools(.).indicators.external_relationships.risk == "high"
+          and any(file.oletools(.).relationships,
+                  .name == "attachedTemplate"
```

```suggestion
                  .name in ("attachedTemplate", "oleObject")
```
We're currently not firing on [this sample](https://app.docguard.io/b5aafc10c38efe0a91c7ceae4b911810c4a148bd55fae301f5283fc1a2779642/c9b82d1a-790f-4cf5-b719-b4f8201d1881/0/results/dashboard), but we would with this change ([analyzer link](https://analyzer.sublime.security?id=ca290aae-5b2c-445a-8b32-5df825bf7a8e)). Thoughts?

---

It _might_ be too loose... `oleObject` is quite broad. We can run some hunts and see what happens?